src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -0,0 +1,170 @@
+    
+    const updateJobDuration = (_, val) => {
+        setJobDuration(val);
+    };
+
+    const updateJobType = value => {
+        setJobType(value.target.value);
+    };
+    
+    return (
+        <Paper 
+            className={searchAreaStyle.searchArea}
+            elevation={8}
+        >
+            <form
+                onSubmit={submitForm}
+                autoComplete="off"
+            >
+                <SearchBar
+                    className={searchAreaStyle.searchBar}
+                    submitSearchForm={submitForm}
+                    searchValue={searchValue}
+                    updateSearchValue={updateSearchValue}
+                />
+                <Collapse 
+                    in={advancedSearch}
+                    classes={{
+                        wrapperInner: classes.wrapperInner
+                    }}    
+                >
+                    <TextField
+                        id="job_type"
+                        select
+                        label="Job Type"
+                        className={searchAreaStyle.jobTypeSelector}
+                        value={jobType}
+                        onChange={updateJobType}
+                        helperText="Please select your job type"
+                    >
+                        {JOB_TYPES.map(({value, label}) => (
+                            <MenuItem
+                                key={value}
+                                value={value}
+                            >
+                                {label}
+                            </MenuItem>
+                        ))}
+                    </TextField>
+                    <FormControl 
+                        className={searchAreaStyle.durationSlider}
+                    >
+                        <Typography 
+                            id="duration-label"
+                            variant="body2"
+                        >
+                            Job Duration
+                        </Typography>
+                        <Slider
+                            valueLabelDisplay="auto"
+                            value={jobDuration}
+                            ValueLabelComponent={SliderValueTooltip}
+                            name='jobDuration'
+                            min={1}
+                            max={12}
+                            step={1}
+                            onChange={updateJobDuration}
+                        />
+                    </FormControl>
+                </Collapse>
+            </form>
+            <ShowAdvancedOptionsButton
+                onClick={toggleAdvancedOptions}
+                open={advancedSearch}
+            />
+        </Paper>
+    );
+}
+
+SearchArea.propTypes = {
+    addSnackbar: PropTypes.func,
+    onSubmit: PropTypes.func.isRequired
+};
+
+const mapStateToProps = () => ({
+});
+
+const mapActionsToProps = {addSnackbar};
+
+export default connect(mapStateToProps, mapActionsToProps)(SearchArea);
```

Comment:
use hooks for consistency?

Reply:
That time will come someday... :P

Reply:
/remind @imnotteixeira in a couple months :V

Reply:
After looking at the Redux hooks docs (https://react-redux.js.org/next/api/hooks) it seems really simple to do the transition, however, they mention some problems that may arise in some edge cases, so I am deferring this refactor to when this solution matures

Reply:
Fair point. Would it make sense to track this in an issue? (Anotha' one)
Probably, I'm guessing